package.json

```diff
@@ -206,5 +206,8 @@
     "utility-types": "2.1.0",
     "which-promise": "1.0.0",
     "write-json-file": "2.2.0"
+  },
+  "dependencies": {
+    "https-proxy-agent": "2.2.1"
```

Comment:
The proxy-agent module should be in `devDependencies` like the other deps.

Reply:
Yeah, The `proxy-agent` module is more powerful and graceful, but it maybe hide more details in my opinion. Need more discussion to choose upon advantages and disadvantages.